R/extract.R

```diff
@@ -90,11 +90,33 @@ extract.Arima <- function(model, include.pvalues = FALSE, include.aic = TRUE,
   return(tr)
 }
 
+#' \code{\link{extract}} method for \code{Arima} objects
+#'
+#' \code{\link{extract}} method for \code{Arima} objects. These objects are
+#' created by the \code{\link[stats]{Arima}} function in the \pkg{stats} package.
+#'
+#' @param model A model object.
+#' @param include.pvalues Report p values in the GOF block?
+#' @param include.aic Report AIC in the GOF block?
+#' @param include.loglik Report log likelihood in the GOF block?
```

Comment:
I think you can inherit the `model`, `include.aic`, and `include.loglik` parameters from `lme4`. Or from `glm` once that is done. (Perhaps you want to squeeze it in before working on the other methods...)